File: BackEnd/postman/Referencia.py
import requests
def postear(data, url):
    return requests.post('http://127.0.0.1/syme/BackEnd/services/'+url+'.php', data=data).content

url='Referencia'
data={
  'accion':'crear',
  'descripcion': None,
  'idCentroMedicoRecibe': None,
  'idExpediente': None,
  'idCentroMedicoRemite': None,
  'idMedico': None,
}
print postear(data,url)
print

url='Referencia'
data={
  'accion':'listarPorPaciente',
  'idPaciente': None,
}
print postear(data,url)
print

url='Referencia'
data={
  'accion':'listarRecibidas',
  'idCentroMedicoRecibe': None,
}
print postear(data,url)
print

url='Referencia'
data={
  'accion':'listarPorCentroRemite',
  'idCentroMedicoRemite': None,
}
print postear(data,url)
print

url='Referencia'
data={
  'accion':'listarPorMedico',
  'idCentroMedicoRemite': None,
  'idMedico': None,
}
print postear(data,url)
print

url='Referencia'
data={
  'accion':'listarTodos',
}
print postear(data,url)
print

url='Referencia'
data={
  'accion':'actualizar',
  'idCentroMedicoRecibe': None,
  'idMedico': None,
  'idReferencia': None,
  'idExpediente': None,
  'idCentroMedicoRemite': None,
  'descripcion': None,
}
print postear(data,url)
print

url='Referencia'
data={
  'accion':'eliminar',
}
print postear(data,url)
print

url='Referencia'
data={
  'accion':'listar',
}
print postear(data,url)
print

File: BackEnd/ajax/Referencia.js
$.ajax({
  url:CONST_SITIO_URL+'/services/Referencia.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'crear',
    'descripcion': null,
    'idCentroMedicoRecibe': null,
    'idExpediente': null,
    'idCentroMedicoRemite': null,
    'idMedico': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Referencia.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorPaciente',
    'idPaciente': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Referencia.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarRecibidas',
    'idCentroMedicoRecibe': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Referencia.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorCentroRemite',
    'idCentroMedicoRemite': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Referencia.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarPorMedico',
    'idCentroMedicoRemite': null,
    'idMedico': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Referencia.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listarTodos',
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Referencia.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'actualizar',
    'idCentroMedicoRecibe': null,
    'idMedico': null,
    'idReferencia': null,
    'idExpediente': null,
    'idCentroMedicoRemite': null,
    'descripcion': null,
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Referencia.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'eliminar',
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


$.ajax({
  url:CONST_SITIO_URL+'/services/Referencia.php',
  method:'POST',
  dataType:'JSON',
  data:{
    'accion':'listar',
  },
  success:function(respuesta){
    console.log(respuesta);
  },
  error: function(error){
    console.log(error);
  },
  complete: function(){
    //TO-DO
  }
});


